src/JobDescription/JobDescription.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import Container from '../Container/Container';
+import { Button } from 'material-ui';
+import { Email } from 'material-ui-icons';
+import Typography from 'material-ui/Typography';
+import { I18n } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { withStyles } from 'material-ui/styles';
+
+const styles = theme => ({
+	button: {
+		margin: theme.spacing.unit,
+	},
+	leftIcon: {
+		marginRight: theme.spacing.unit,
+	},
+});
+
+function hashTagify(string) {
+	let words = string.split(' ');
+	let capitalized = words.map(
+		word => word.charAt(0).toUpperCase() + word.substr(1)
+	);
+	let hashTag = `#${capitalized.join('')}`;
+	return hashTag;
+}
+
+const JobDescription = props => (
```

Comment:
Can it be use a decorator as in `Intro.jsx #47`?

Reply:
I'll see what can be done (not familiar with all this stuff :D especially not JS/react decorators)

Reply:
For this one, I don't think we can use a decorator, since the translation namespace needed is in the props and this component dynamically uses a different translation file depending on which job you're showing:

```jsx
<I18n ns={[`${props.translation}`, 'jobs']}>
```

Since the decorator is outside of the class:

```js
@translate(['intro'], { wait: true })
class Intro extends Component {
```

We don't have access to the props here 🤔